R/helpers.R

```diff
@@ -262,7 +262,7 @@ dinvGamma <- function(x,
 ##' @param b the scale parameter of the inverse gamma distribution
 ##' @param lower.tail logical; if TRUE (default), probabilities are `P(X  > x)`,
 ##'   otherwise, `P(X <= x)`.
-##' @param logical; FALSE (default) if TRUE, probabilities/densities p
+##' @param logical FALSE (default) if TRUE, probabilities/densities p
```

Comment:
```suggestion
##' @param logical whether probabilities/densities p
```

Reply:
(generally we don't want to list default values in the argument documentation)

Reply:
Corrected parameter name and made the suggested change.